# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/esden/dvorakng/install_path.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./Makefile.orig	2003-04-25 23:52:48.000000000 +0200
+++ ./Makefile	2003-08-01 11:19:27.000000000 +0200
@@ -1,4 +1,4 @@
-INSTALL = /usr/local/games/
+INSTALL = $(prefix)/bin/
 
 # Uncomment these lines if you want to change the defaults to something else
 # (This is/will be removed - and better mechanism will be used for this purpose - Nopik)
